### PR TITLE
added backend routes

### PR DESCRIPTION
**Pull request by jspicuzza21**
Added backend things.

### CODE REVIEW

**Review by kwons90 — APPROVED** [review]
All hands on deck

File: src/client/components/home.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { FC } from 'react';
 
-const Home : FC = () => {
+const Home:FC = () => {
```
Comment (chadalad) [inline src/client/components/home.tsx:3]: We can double check this syntax

File: src/client/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDom from 'react-dom';
 import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import { Home } from './components/index'
+import Home from './components/home.tsx'
```
Comment (chadalad) [inline src/client/index.js:5]: we need to come up w/ a norm once we know how to handle this situation

File: src/server/api/routes/userRoutes.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+const { Router } = require('express');
+const bcrypt = require('bcrypt');
+const { models:{ User, Session } }= require('../../db/index');
+
+const apiRouter = Router();
+
+apiRouter.post('/login', async (req, res)=>{
```
Comment (chadalad) [inline src/server/api/routes/userRoutes.js]: as discussed change to userRouter

**Review by chadalad — CHANGES_REQUESTED** [review]
quick change to make apiRouter userRouter

**Pull request merged**